inst/include/Rcpp/api/meat/message.h

```diff
@@ -0,0 +1,33 @@
+
+// message.h: Rcpp R/C++ interface class library -- Wrapper for base::message
+//
+// Copyright (C) 2021  Dirk Eddelbuettel
+//
+// This file is part of Rcpp.
+//
+// Rcpp is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Rcpp is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Rcpp.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef Rcpp_api_meat_message_h
+#define Rcpp_api_meat_message_h
+
+namespace Rcpp {
+
+    inline void message(SEXP s) {
```

Comment:
If the goal is to complete the trio `message`, `warning`, `stop`, shouldn't Rcpp provide a string-based interface as well as a variadic one (format string + args), as `warning` and `stop` do?

Reply:
Just in case you didn't see this one. :point_up:

Reply:
I did see it.  Minimal does it for now, for me at least.

Reply:
`Rcpp::message()` is a band aid, and we agreed that a band aid is good enough. I don't feel compelled to dress it up.

As always, the sky is limit, but incremental approaches work.  If we think this one is good to merge, we merge it.  Afterwards is someone (you?) feels like expanding we can discuss that question and approach.